test/test.ts

```diff
@@ -25,12 +27,13 @@ test('udp-device-connects', (t) => {
     'A5A5A5A5'
   );
 
+  // Mock a UDP Network
   const mockNetwork = new MockNetwork();
 
   // Mock the Local Home Platform
-  const mockLocalHomePlatform = new MockLocalHomePlatform(mockNetwork, [
-    scanConfig,
-  ]);
+  const mockLocalHomePlatform = MockLocalHomePlatform.getInstance();
+
+  mockLocalHomePlatform.initializeRadio(mockNetwork, [scanConfig]);
```

Comment:
what about simplifying this (#27) rather than introducing a new public interface element.

Reply:
Will address in #27